packages/ui5-library-inquirer/test/unit/prompts/prompts.test.ts

```diff
@@ -13,6 +13,13 @@ jest.mock('@sap-ux/project-input-validator', () => {
     };
 });
 
+jest.mock('@sap-ux/inquirer-common', () => {
```

Comment:
Can you explain to me why we need this?

Reply:
Its a workaround for : https://stackoverflow.com/questions/67872622/jest-spyon-not-working-on-index-file-cannot-redefine-property

Reply:
Thanks, didn't know that one.